vllm/model_executor/models/mixtral.py

```diff
@@ -171,6 +203,12 @@ def __init__(self,
         self.rope_theta = rope_theta
         self.sliding_window = sliding_window
 
+        if isinstance(linear_method, Fp8LinearMethod):
+            # FIXME(pcmoritz): If we are using FP8, we currently do
+            # not want to quantize the attention layers until we improve
+            # the performance and make sure the accuracy is good.
+            linear_method = None
```

Comment:
Nit: It would be nice to have this as a log since my assumption is all Linear modules would be quantized like in the initial FP8 PR, but I assume this will be a fast follow to fix

Reply:
Fixed :)